### PR TITLE
add docs related to getting started

### PR DESCRIPTION
**Pull request by bfan1256**
This adds documentation to help improve user onboarding with Blankly

### CODE REVIEW

File: content/en/getting-started/features.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    - Customizable circular buffer websocket feeds
+    - Support for Coinbase Pro & Binance sandbox modes
+    - Instant paper trade wrapper for supported exchanges
+    - Run scheduled functions natively
+    - Logs for websocket feeds
+    - Status management for purchases
+    - Create strategies using an event-based framework
+    - Instantly backtest your strategies (`strategy.backtest(to='1y')`)
+    - Automatic price downloads & caching for use in backtests
+    - Interface that allows calls to each supported exchange to be identical
```
Comment (EmersonDove) [inline content/en/getting-started/features.md:29]: Love how the feature list is kept. This is a way better spot for it.

File: content/en/index.md [hunk 2]
```diff
@@ -1,42 +1,109 @@
...
+
+<alert type="success">
+Our goal is to take what originally takes 3-4 months of scrounging for data, connecting endpoints, and more, and make it only a few lines of code so that your workflow is optimized and easy
+</alert>
+
+
+### One Codebase, Any Exchange, Trading Multiple Entities
+
+Blankly is the only package on the market that allows you to build a model once and allow it to run on any supported exchange and trading multiple entities at a time.
+You can trade `MSFT`, `AAPL`, `PLTR` all from a single code-file with three lines of code.
```
Comment (EmersonDove) [inline content/en/index.md:22]: 3 line claim may be aggressive

Comment (EmersonDove) [inline content/en/index.md:22]: 3 line claim may be pushing it

Comment (EmersonDove) [inline content/en/index.md:22]: 3 line claim may be pushing it

File: content/en/index.md [hunk 3]
```diff
@@ -1,42 +1,109 @@
...
+    history = interface.get_product_history(50, '1d')
+
+    # easily integrate your model
+    decision = MyAwesomeModel(history)
+
+    # buy or sell based on that decision
+    if decision:
+        interface.market_order('buy', 0.25 * interface.cash)
+        state.variables['has_buy_order'] = true
+    else if state.variables['has_buy_order'] && !decision:
```
Comment (EmersonDove) [inline content/en/index.md:52]: Could increase the pythonicness of the comparisons

File: content/en/index.md [hunk 4]
```diff
@@ -1,42 +1,109 @@
...
+    interface: Blankly.Interface = state.interface
+    # get last 50 days worth of price data 
+    history = interface.get_product_history(50, '1d')
+
+    # easily integrate your model
+    decision = MyAwesomeModel(history)
+
+    # buy or sell based on that decision
+    if decision:
+        interface.market_order('buy', 0.25 * interface.cash)
```
Comment (EmersonDove) [inline content/en/index.md:50]: I would like to have an implementation of this before pushing or advertising

**Review by EmersonDove — COMMENTED** [review]
Looks great. I would like to add the features you use to the code before pushing to the site, maybe use available as a property so that all the code works.

File: content/en/index.md (hunk 2 above)
Comment (bfan1256) [reply content/en/index.md:22]: Maybe three additional lines? Was just trying to say that it's as simple as 
```python
s.add_price_event(..., 'PLTR')
s.add_price_event(..., 'MSFT')
s.add_price_event(..., 'AAPL')

Could use some rewording
```

**Pull request merged**